### ISSUE TITLE
Automatic export to file

### ISSUE THREAD

**Issue by Markus40** [Livinglist/Hacki#431]
Exporting data manually on a regular basis is tedious. It would be nice to have an automatic export to a file.

**Comment by Livinglist** [comment]
Hi @Markus40 are you talking about exporting your favorites? Where do you want it to be automatically saved to?

**Comment by Markus40** [comment]
Yes. Some applications allow you to select a directory in the file system to which they periodically export a backup. The interface often looks the same, so I suspect there is a default lib (or something similar for Android development) for choosing a specific directory.
